.eslintrc.js

```diff
@@ -19,6 +19,10 @@ module.exports = {
       'indent': [2, 4],
       'no-underscore-dangle': 'off',
       'camelcase': 'off',
-      'no-unused-vars': 'off'
+      'no-unused-vars': 'off',
+      'linebreak-style': 'off',
+      'arrow-parens': 'off',
+      'consistent-return': 'off',
+      'class-methods-use-this': 'off'
```

Comment:
Please do not yet change these, We should discuss them one by one.

Reply:
I will describe these changes:
`'linebreak-style': 'off'` - I'm working on windows and got 300 warnings after linting that "Expected linebreaks to be 'LF' but found 'CRLF'", so I think this is a nice a idea to disable this rule at all, than check linebreaks between different OS. Also I do not thing that it somehow affect code quality.
'arrow-parens': 'off' - this allows us to write a callback function with one argument without wrapping parentheses, for me it is quite clean. In addition, the code has already been written in this way, so I believe that this rule can be omitted.
'consistent-return': 'off' - there are many cases when you do not want to return anything from the callback function, you just need to modify an array, I believe that this is unnecessarily strict rule.
'class-methods-use-this': 'off' - eslint shows an error if you have some method of the class where you do not use `this`. And you can make this method static for example or switch off this eslint rule. I also think that with disabled rule the code is cleaner.

Reply:
> 'linebreak-style': 'off'

I will test this on my Windows machine.

We can set this in the .gitattributes file and .editorconfig too.

No need to disable it ;-)

Reply:
> 'arrow-parens': 'off'

We should be concise and keep it on =)

Reply:
> 'consistent-return': 'off'

We should keep that on to see where nothing is returned. This will be important for the unit tests and async / await and promised based solution. There should be always returned something.

The callback based approach is currently using an inconsistent style.

Reply:
> 'class-methods-use-this': 'off'

We can easily resolve this and ensure that the client will be more configurable.

Otherwise we should use the `eslint-disable` comments. Please don't globally disable a rule so many issues will not be found.

See https://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments